EcalTiming/test/ecalTime_fromAlcaStream_cfg.py

```diff
@@ -23,9 +33,15 @@
                  "0=false, 1=true"
                  )
 ### setup any defaults you want
-options.output="ecalTiming.root"
+streamName = "AlCaPhiSym"
```

Comment:
Yes, we tried, no success (but didn't investigated much, wanted something running fast)